### PR TITLE
Release/1.5.0

### PR DESCRIPTION
**Pull request by vincent-gao**
### New features:

- Drupal 8.7.9 !
- tide_workflow_notification module
- vertical workflow form in node view
- ClamAV uses local executable way in local and CI
### Bugs fix
- drupal/scheduled_transitions
  - [The revision user is incorrect after CRON ran](https://www.drupal.org/project/scheduled_transitions/issues/3094322)
- drupal/viewsreference
  - [Updating from alpha4 to alpha5 via composer results in error](https://www.drupal.org/project/viewsreference/issues/3096956#comment-13370643)

### UX
- Make the DateTime input format and descriptive text consistent

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "drupal/config_ignore": "^2.1",
         "drupal/config_split": "^1.4",
         "drupal/config_update": "^1.6",
-        "drupal/core": "8.6.16",
+        "drupal/core": "8.7.9",
```
Comment (anthony-malkoun) [inline composer.json]: 👍

File: composer.json [hunk 2]
```diff
@@ -45,12 +45,12 @@
...
         "drupal/simple_sitemap": "^2.12",
         "drupal/smart_trim": "^1.1",
-        "drupal/stage_file_proxy": "^1.0-alpha3",
+        "drupal/stage_file_proxy": "1.0-rc2",
         "drupal/token_conditions": "dev-1.x",
         "drupal/token_filter": "^1.1",
         "drupal/twig_field_value": "^1.1",
         "drupal/views_menu_children_filter": "^1.0-beta2",
-        "drupal/viewsreference": "^2.0-alpha4",
+        "drupal/viewsreference": "2.0-alpha5",
```
Comment (anthony-malkoun) [inline composer.json]: 👍

File: composer.json [hunk 3]
```diff
@@ -69,9 +69,7 @@
             "drupal/core": {
                 "Contextual links should not be added inside another link - https://www.drupal.org/node/2898875": "https://www.drupal.org/files/issues/2018-03-16/contextual_links_should-2898875-6.patch",
                 "RSS view with fields give wrong URL from path field - https://www.drupal.org/node/2673980#comment-12180229": "https://www.drupal.org/files/issues/rss_view_with_fields-2673980-28.patch",
-                "DateTimeIso8601::getDateTime() does not specify the storage timezone when constructing DrupalDateTime object - https://www.drupal.org/project/drupal/issues/3002164": "https://www.drupal.org/files/issues/2018-09-25/3002164-7.patch",
                 "Custom blocks are no longer displayed and no longer appear in the custom block library - https://www.drupal.org/project/drupal/issues/2997898#comment-12832813": "https://www.drupal.org/files/issues/2018-10-27/2997898-21.patch",
-                "Make the DateTime input format and descriptive text consistent - https://www.drupal.org/project/drupal/issues/2791693": "https://www.drupal.org/files/issues/2018-06-22/2791693-datetime-input-ui-29.patch",
```
Comment (anthony-malkoun) [inline composer.json]: @vincent-gao I assume this didn't apply... did we check if there was an updated patch or is it no longer required?

Comment (vincent-gao) [reply composer.json]: 1. 
```
"Make the DateTime input format and descriptive text consistent - https://www.drupal.org/project/drupal/issues/2791693": "https://www.drupal.org/files/issues/2018-06-22/2791693-datetime-input-ui-29.patch"
```
was committed to drupalcore 8.7.x no need for this patch.

**Pull request merged**